src/main/java/com/hubspot/jinjava/lib/filter/IpAddrFilter.java

```diff
@@ -52,26 +71,66 @@ public Object filter(Object object, JinjavaInterpreter interpreter, String... ar
       return false;
     }
 
-    if (args.length > 0) {
-      String function = args[0].trim();
-      return getFunctionValue(interpreter, function.toLowerCase(), object);
+    String parameter = getParameter(args);
+    if (object instanceof List) {
```

Comment:
Should we expand this to other iterables supported by Jinjava? Generally filters will use https://github.com/HubSpot/jinjava/blob/master/src/main/java/com/hubspot/jinjava/util/ObjectIterator.java#L36 to cover some edge cases. Can we add a static method there to check if this is a list/array/iterable etc. and then use `getLoop` to get a more general `Iterator`?

Reply:
I changed it from List to Iterables and added support for filtering Map objects.
But I don't think that it's worth to add static method for checking if object is Iterable or not because it's just one line with 'instanceof' operator.